sdk/Lusid.Sdk.Tests/TokenProviderTests.cs

```diff
@@ -28,6 +28,19 @@ public async Task CanGetToken()
             // THEN it is populated
             Assert.That(token, Is.Not.Empty);
         }
+        
+        [Test]
+        public async Task CanGetPACToken()
```

Comment:
sorry to be a pedant, this should either be `PAT` or `PersonalAccessToken` (I prefer the later for explicitness and readability)